L1Trigger/L1TMuon/BuildFile.xml

```diff
@@ -5,29 +5,32 @@
   <use name="xerces-c"/>
 </export>
 
-<use name="L1Trigger/L1TCommon"/>
+
+<use name="root"/>
+<use name="rootmath"/>
+<use name="rootrflx"/>
+<use name="rootgraphics"/>
```

Comment:
are rootgraphics and root really needed here?

Reply:
Removed.